Library/Contributions/example-formula.rb

```diff
@@ -165,7 +168,7 @@ def pour_bottle?
   depends_on :hg # Mercurial (external or brewed) is needed
 
   # If any Python >= 2.7 < 3.x is okay (either from OS X or brewed):
-  depends_on :python
+  depends_on :python if MacOS.version <= :snow_leopard
```

Comment:
This is correct without the conditional. We use the conditional in formulas where we otherwise use system Python because we always assume we have Python 2.7 and system Python is older on 10.6 and before, and :python is guaranteed to give you 2.7.

Maybe leave this line as it was and add another line below, like:
```
# to depend on Python >= 2.7 but use system Python where possible
depends_on :python if MacOS.version <= :snow_leopard
```